src/scripts/main.js

```diff
@@ -1,3 +1,49 @@
 'use strict';
 
-// write your code here
+function promise1() {
+  return new Promise((resolve, reject) => {
+    const logo = document.getElementById('logo');
+
+    logo.addEventListener('click', () => {
+      const successDiv = document.createElement('div');
+
+      successDiv.classList.add('message');
+      successDiv.textContent = 'Promise was resolved!';
+      document.body.appendChild(successDiv);
+
+      resolve();
+    });
+  });
+}
+
+function promise2() {
+  return new Promise((resolve, reject) => {
+    setTimeout(() => {
+      const errorDiv = document.createElement('div');
+
+      errorDiv.classList.add('message');
+      errorDiv.classList.add('error-message');
+      errorDiv.textContent = 'Promise was rejected!';
+      document.body.appendChild(errorDiv);
+
+      reject(new Error('Promise was rejected!'));
+    }, 3000);
+  });
+}
+
+promise1()
```

Comment:
here also we can now simplify our promises calls just to be:

```
promise1
  .then(onSuccess)
  .catch(onError);

promise2
  .then(onSuccess)
  .catch(onError);
```

we pass here our new created functions to methods .then and .catch

Reply:
Thank you very much.